CHANGELOG.asciidoc

```diff
@@ -70,6 +70,8 @@ https://github.com/elastic/beats/compare/v6.0.0-beta2...master[Check the HEAD di
 
 *Filebeat*
 
+- Right check of module/fileset exists for create-fileset command {issue}5657[5657]
```

Comment:
@kvch I think we should remove these lines from the changelog again as we normally don't up dev related stuff in here. Perhaps we should put it in an other category? We discussed it in the past where to put it but never came to a conclusion :-(

Reply:
@ruflin do you have a link on discussion?